secrets/secrets.yaml

```diff
@@ -1,12 +1,12 @@
 secrets:
-  install_me_password: F7t^s%xTO64)
+  installMePassword: F7t^s%xTO64)
```

Comment:
nit: Maybe change is to `Pa$$w0rd` to make it more clear; this is an example.

Reply:
@cfernhout  : I'll do that in another pr